### ISSUE TITLE
`consistent-list-newline` false positive in `import` statement

### ISSUE THREAD

**Issue by xsjcTony** [antfu/eslint-plugin-antfu#22]
### Describe the bug

No idea whether this is actually a false positive, but I think this format is definitely valid, as I can't think of a better way to mix default import and name imports:
![image](https://github.com/antfu/eslint-plugin-antfu/assets/45784210/156c7f76-4472-4907-afd1-937d19291087)
```ts
import Icon, {
  MailOutlined,
  NumberOutlined,
  QuestionCircleOutlined,
  QuestionOutlined,
  UserOutlined,
} from '@ant-design/icons';
```

### Reproduction

See above

### System Info

```Shell
N/A
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)
